reference/implementing-a-concept-exercise.md

```diff
@@ -18,7 +20,8 @@ As this document is generic, the following placeholders are used:
 - `<NAME>`: the name of the exercise (e.g. `Calculator Conundrum`).
 - `<CONCEPT_SLUG>`: the slug of one of the exercise's concepts in kebab-case (e.g. `anonymous-methods`).
 
-Before implementing the exercise, please make sure you have a good understanding of what the exercise should be teaching (and what not). This information can be found in the exercise's GitHub issue.
+Before implementing the exercise, please make sure you have a good understanding of what the exercise should be teaching (and what not).
+This information can be found in the exercise's GitHub issue.
```

Comment:
Not sure this should be two paragraphs.

Reply:
Sorry, was reviewing this in the graphical browser, and it is not what it seemed to be.  Disregard.